corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -655,7 +653,13 @@ def sync_log(self):
 
     @property
     def async_cache_key(self):
-        return restore_cache_key(self.domain, ASYNC_RESTORE_CACHE_KEY_PREFIX, self.restore_user.user_id)
+        return restore_cache_key(
```

Comment:
async restore stores the ID of running tasks in the cache. If there is a task currently running with that ID it returns a 202 with the current progress of the sync. 
Adding device ID here means that if you start a restore on one device and then try and restore on a different one while the task is running, it will create 2 tasks, which I think might be what you want. If the cache of that key is deleted, it won't find the task, so will kick off a new one. 
It should be fine to add the synclog_id too.